ansible/roles/postgres-init/tasks/main.yml

```diff
@@ -7,7 +7,7 @@
 
     - name: Postgres                   | Add PostgreSQL YUM repository
       yum_repository:
-        name: percona-ppg-11-experimental
+        name: percona-ppg-11-release
```

Comment:
I guess we can it rename to `percona-ppg-11`. That way one can change a variable to `experimental` and not be surprised

Reply:
ok